cluster/gce/gci/configure-helper.sh

```diff
@@ -2013,6 +2013,26 @@ function setup-coredns-manifest {
   sed -i -e "s@{{ *pillar\['service_cluster_ip_range'\] *}}@${SERVICE_CLUSTER_IP_RANGE}@g" "${coredns_file}"
 }
 
+function setup-fluentd {
+  setup-addon-manifests "addons" "fluentd-gcp"
+  local -r event_exporter_yaml="${dst_dir}/fluentd-gcp/event-exporter.yaml"
```

Comment:
`dst_dir` is a local variable of the function you extracted this from

Please pass variables via function parameters

Reply:
If this causes this to be broken, the same could apply to `setup-coredns-manifest`, and potentially other helpers that exist throughout this file.